main.go

```diff
@@ -484,8 +486,38 @@ func (carina *ClusterCommand) Get(pc *kingpin.ParseContext) (err error) {
 }
 
 // Delete a cluster
-func (carina *ClusterCommand) Delete(pc *kingpin.ParseContext) (err error) {
-	return carina.clusterApply(carina.ClusterClient.Delete)
+func (carina *CredentialsCommand) Delete(pc *kingpin.ParseContext) (err error) {
+	err = carina.clusterApply(carina.ClusterClient.Delete)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Unable to delete cluster, not deleting credentials on disk")
+		return err
+	}
+	p, err := carina.clusterPath()
+
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Unable to locate carina config path, not deleteing credentials on disk\n")
+		return err
+	}
+
+	// TODO: Any other failsafes we should check here?
+	if p == "" || p == "/" {
```

Comment:
If you stick with `path.Clean`, which you are using everywhere else, then root is "/". I am just unsure if `ioutils` is smart enough to accept paths made by `path` and turn them into Windows paths at the last minute? I assume so since carina mostly works on windows already!

Let me know if you need me to help test out what go does in these cases on Windows.